userspace/libscap/engine/gvisor/parsers.cpp

```diff
@@ -70,7 +70,7 @@ static void fill_context_data(scap_evt *evt, T& gvisor_evt)
 
 static parse_result parse_container_start(const char *proto, size_t proto_size, scap_sized_buffer scap_buf)
 {
-	struct parse_result ret;
+	parse_result ret;
```

Comment:
I know this is not strictly necessary but I would love to have this initialized

```suggestion
	parse_result ret = {0};
```

Same for other cases...